src/Runner.js

```diff
@@ -90,11 +93,10 @@ function run(transformFile, paths, options) {
     return;
   }
 
-  getAllFiles(paths)
-    .then(files => files.filter(
-      name => !extensions || extensions.indexOf(path.extname(name)) != -1
-    ))
-    .then(files => {
+  getAllFiles(
+    paths,
+    name => !extensions || extensions.indexOf(path.extname(name)) != -1
```

Comment:
wait, no, you are not right.

extensions is an array and the value passed to indexOf is the extension name of the file.